LogicalTypes.md

```diff
@@ -599,6 +599,45 @@ optional group variant_shredded (VARIANT) {
 }
 ```
 
+### GEOMETRY
+
+`GEOMETRY` is used for geometry features in the Well-Known Binary (WKB) format
+with linear/planar edges interpolation. It must annotate a `BYTE_ARRAY`
+primitive type. See [Geospatial.md](Geospatial.md) for more detail.
+
+The type has two type parameters:
+- `crs`: An optional string value for CRS. If unset, the CRS defaults to
+  "OGC:CRS84", which means that the geometries must be stored in longitude,
+  latitude based on the WGS84 datum.
+- `crs_encoding`: An optional enum value to describes the encoding used by the
```

Comment:
In the Iceberg PR, the `crs` and `crs_encoding` are merged to a single `crs` field and formatted as ` $type:$content`, where `type` allows `srid` and `projjson`. Should we follow the same?

Reply:
This is the only thing that I am hesitant to keep it consistent with the Iceberg PR. The mapping between them follows the table below:

| Iceberg | Parquet | 
|----------|----------|
| `$type`    | `crs_encoding` | 
| `$content`  | `crs` | 

The reason is to provide a chance for non-Iceberg use cases to write full projjson content to the `crs` field. But this is not a strong opinion. Let me know what you think.

Reply:
Makes sense! I think the current way in Parquet is better and we can keep it.

Reply:
I'm strongly against the current type parameters. The CRS should not be embedded, it should be referenced like the spec in Iceberg. There's no need to duplicate these, require support for specific formats, or make type annotations huge. A CRS reference ID is cleaner, along with a way to store the CRS definition in file metadata if that's desired.

Reply:
I've changed the `crs` field to be a simple string without restriction. Let me know what you think. @rdblue @paleolimbot